misk/src/main/kotlin/misk/web/metadata/AdminDashboardModule.kt

```diff
@@ -30,63 +31,66 @@ class AdminDashboardModule(val environment: Environment) : KAbstractModule() {
     install(WebActionModule.create<AdminDashboardTabAction>())
     install(WebActionModule.create<ServiceMetadataAction>())
     install(WebTabResourceModule(
-        environment = environment,
-        slug = "loader",
-        web_proxy_url = "http://localhost:3100/"
+      environment = environment,
+      slug = "loader",
+      web_proxy_url = "http://localhost:3100/"
     ))
     install(WebTabResourceModule(
-        environment = environment,
-        slug = "loader",
-        web_proxy_url = "http://localhost:3100/",
-        url_path_prefix = "/_admin/",
-        resourcePath = "classpath:/web/_tab/loader/"
+      environment = environment,
```

Comment:
this reformatting is wrong

Reply:
Our KtLint configuration changes the `continuation_indent_size` from the Kotlin default 4 to 2:

```
ktlint(dep.ktlintVersion).userData(['indent_size': '2', 'continuation_indent_size' : '2'])
```